cases/models.py

```diff
@@ -427,26 +427,28 @@ def merged_into_final(self):
     @cached_property
     def timeline_merge_records(self):
         """Return a list of MergeRecords to display in the timeline for this case."""
-        merge_map = self.merge_map
-        mergee_records = MergeRecord.objects.filter(
-            Q(merged_into_id__in=merge_map.keys()) | Q(mergee_id__in=merge_map.keys())
+
+        # Records this case is is referenced in.
+        direct_records = MergeRecord.objects.filter(
+            Q(merged_into_id=self.id) | Q(mergee_id=self.id)
         ).all()
 
-        merged_into_list = self.merged_into_list
-        merged_into_ids = [m["id"] for m in merged_into_list]
-        merged_into_records_query = Q(merged_into_id__in=merged_into_ids) | Q(
-            mergee_id__in=merged_into_ids
+        # Records referencing cases that are currently merged into this case.
+        merge_map = self.merge_map
+        upstream_records = MergeRecord.objects.filter(
+            Q(merged_into_id__in=merge_map.keys()) | Q(mergee_id__in=merge_map.keys())
         )
-        for merged in merged_into_list:
-            merged_into_records_query |= Q(time__gte=merged["at"]) & (
+
+        # Records referencing cases that this case has merged into (possibly transitively),
+        # after the time of the merge.
+        downstream_records_query = Q(pk=None)  # Match nothing.
```

Comment:
You can use an empty `Q()` for the same effect.

Reply:
`Q()` appears to match _everything_, even when it's negated.

Reply:
oh, my bad, sorry!